CHANGELOG.md

```diff
@@ -1,14 +1,36 @@
 **New Features and Improvements**
-* Add direct aiming from Crispy Doom. More accurate vertical aiming (thanks to @ceski-1).
-* Optimizations for plane rendering, taken from Eternity Engine (thanks to @JNechaevsky).
-* Add `-dedicated`, `-uncapped` and `-nouncapped` command line parmeters (thanks to @loopfz).
-* Rename `-nodehlump` to `-nodeh` for consistency with other ports.
-* Use sector lightlevel for sprites in Boom and Vanilla complevels.
+* 3D audio support (@ceski-1)
+  - HRTF mode ("Headphones mode" in the General menu).
+  - Air absorption and Doppler effects settings in the config.
+* PC Speaker emulation sound module (taken from Chocolate Doom).
+* Various HUD additions and fixes:
+  - Optional widescreen widget arrangement.
+  - Bring back three-lined coords/stats widgets.
+  - Optionally draw bar graphs in Boom HUD widgets.
```

Comment:
@fabiangreffrath I don't really understand what exactly is fixed/added to the HUD, there seems to be a lot of changes! Please give your input on this.

Reply:
Ability to position "message" and "secret" text widgets by WOOFHUD lump.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1208/commits/465b0e9c0aa1bad60b27c9a7e590a9538aa78e2b